scss/bitstyles/ui/dropdown-menu.stories.mdx

```diff
@@ -29,18 +29,43 @@ For the sake of clarity, the examples shown here are all presented with the menu
   </ul>
 </details>
 
+## Default
+
+These examples show various contents for the dropdown menus — what you put in it is up to you, but things to note are:
+
+- the `<li role="separator"></li>` that provide visual and semantic sepearators
+- the standard menu items are likely to be created using `.a-button.a-button--menu`
+
 <Canvas>
-  <Story name="Dropdown menu" inline={false} height="20rem">
+  <Story name="Dropdown menu">
     {`
-      <div style="min-height:15rem;">
-        <div class="u-relative">
-          <button type="button" class="a-button a-button--ui u-h6" aria-controls="dropdown-1" aria-expanded="true">
+      <div style="min-height:20rem;">
+        <div class="u-relative" x-data="{ dropdownOpen: false }">
+          <button
+            type="button"
+            class="a-button a-button--ui u-h6"
+            aria-controls="dropdown-1"
+            @click="dropdownOpen = !dropdownOpen"
+            :aria-expanded="dropdownOpen"
+          >
             <span class="a-button__label">Account</span>
             <svg width="14" height="14" class="a-button__icon a-icon a-icon-m" aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
               <use xlink:href="${icons}#icon-caret-down"></use>
             </svg>
           </button>
-          <ul class="a-dropdown u-overflow--y a-list-reset u-margin-s-top" aria-hidden="false" id="dropdown-1">
+          <ul
+            x-show="dropdownOpen"
+            x-transition:enter="is-transitioning"
+            x-transition:enter-start="is-off-screen"
+            x-transition:enter-end="is-on-screen"
+            x-transition:leave="is-transitioning"
+            x-transition:leave-start="is-on-screen"
+            x-transition:leave-end="is-off-screen"
```

Comment:
That's for Alpine.js, right? Wondering if we should mention that, so that people don't copy everything into their code 🤔

Reply:
Yes good point. 👍 I’ll flag it in the intro to each of the components, and add html comments next to each of these blocks

Reply:
I added a banner to the intro of each of the JS-using pages, but I can’t get html comments working in the context of storybook stories. I’ll add an issue and come back to it